src/commands/execute.ts

```diff
@@ -3,7 +3,17 @@ import { inquireConfirmationStep } from "../modules/inquiry/inquireConfirmationS
 import { logger } from "../logger";
 import { getRunbookList } from "../modules/runbook/runbookList";
 
-export const handler = async () => {
+const enum Commands {
+  Init = "execute",
+}
+const enum Package {
+  Description = "Run each of the runbooks on selected AWS instance",
+}
+
+export const command = [Commands.Init];
```

Comment:
Why are we exporting this? Couldn't find where this is being used (at least not in this PR).

Reply:
It's for yargs to [provide a command module](https://github.com/yargs/yargs/blob/master/docs/advanced.md#providing-a-command-module). I use it to define the command and helper message.